### PR TITLE
Filter batch changes by submitter and date time range

### LINKED ISSUE #1245: Add Search and Filter to DNS Changes All Requests tab
**Describe what you'd like to see added or improved in VinylDNS**
Currently it's very difficult to track down a specific DNS change in the All Requests tab if you don't know the change ID. Being able to search/filter by submitter and datetime range would make this much easier. We could also add a filter all the change statuses to find failed and pending processing changes.

### PR DESCRIPTION
**Pull request by Aravindh-Raju**
Partially fixes #1245.

Changes in this pull request:
- Super users can now filter DNS Changes by submitter's username in `All Requests` tab.
- Super users can now filter DNS Changes by date time range in `All Requests` tab.

### CODE REVIEW

File: modules/portal/app/views/recordsets/recordSets.scala.html
Comment (nspadaccino) [inline modules/portal/app/views/recordsets/recordSets.scala.html:1]: are the changes here just to make the recordset search box and the dnschanges search box visually consistent?

File: modules/portal/public/lib/dns-change/dns-changes.controller.js [hunk 1]
```diff
@@ -55,7 +55,7 @@
                 }
 
                 return dnsChangeService
-                    .getBatchChanges(batchChangePaging.maxItems, undefined, $scope.ignoreAccess, $scope.approvalStatus)
+                    .getBatchChanges($scope.submitterName, batchChangePaging.maxItems, undefined, $scope.ignoreAccess, $scope.approvalStatus)
```
Comment (nspadaccino) [inline modules/portal/public/lib/dns-change/dns-changes.controller.js:58]: is this scope created in the dnsChanges.scala.html view?

File: modules/portal/app/views/recordsets/recordSets.scala.html
Comment (Aravindh-Raju) [reply modules/portal/app/views/recordsets/recordSets.scala.html:1]: Yeah. Record search filter box didn't have proper margin like other filters, so added them to it.

File: modules/portal/public/lib/dns-change/dns-changes.controller.js (hunk 1 above)
Comment (Aravindh-Raju) [reply modules/portal/public/lib/dns-change/dns-changes.controller.js]: Yes, it is created with the search box. `ng-model="submitterName"`.

**Review by nspadaccino — DISMISSED** [review]
Looks good to me, filtering works as expected using prod snapshot db

**Review by arpit4ever — COMMENTED** [review]
LGTM
Filtering works on local & prod env.

**Review by nspadaccino — APPROVED** [review]
Looks good, datetime picker is aligned

**Review by arpit4ever — APPROVED** [review]
LGTM
date time picker looks good now.

**Pull request merged**